bin/main.ml

```diff
@@ -685,12 +685,17 @@ let get_prefix context ~from_command_line =
   | Some p -> Future.return (Path.of_string p)
   | None -> Context.install_prefix context
 
+let get_libdir context ~from_command_line =
+  match from_command_line with
+  | Some p -> Future.return (Path.of_string p)
+  | None -> Context.install_ocaml_libdir context
```

Comment:
I think the default for `--libdir` should be `$prefix/lib`. This is what `opam-installer` does. Otherwise, if you call `jbuilder install --prefix ...`, it might install library files at a completely different location